src/DssSpell.t.sol

```diff
@@ -266,7 +266,7 @@ contract DssSpellTest is DSTest, DSMath {
             aL_gap:       100 * MILLION,
             aL_ttl:       12 hours,
             line:         0 * MILLION,     // Not checked as there is auto line
-            dust:         2000,
+            dust:         5000,
```

Comment:
This test might fail as ETH-C is not part of the ilk registry yet. Testing...

Reply:
There is a section to execute previously scheduled spells so we are good.

Reply:
The test is set to cast the previous spell first, so we're good here.

Reply:
All the tests passed. What am I missing?

Reply:
Right, OK, thanks